src/configuration/app/variables.md

```diff
@@ -0,0 +1,115 @@
+# Variables
+
+Platform.sh provides a number of ways to set [variables](/development/variables.md), either globally or specific to a single environment.  For values that should be consistent between different environments (because they're configuring the application or runtime itself, generally) the easiest way to control them is to set them in the `.platform.app.yaml` file.
+
+Only prefixed variables may be set from the `.platform.app.yaml` file.  Some prefixes have specific meaning while others are only significant to a particular application.  Nested variables will be automatically converted into a nested array or list structure as appropriate to the language.
```

Comment:
Yes.  Only variables with a colon prefix.  Are you saying it's unclear what it means, or "seriously, we only support prefixed vars"?